src/main.qml

```diff
@@ -1095,11 +1113,13 @@ ApplicationWindow {
                             Layout.fillWidth: true
                             font.family: roboto.name
                             font.pointSize: 12
-                            color: "grey"
+                            color: !dstitem.unselectable ? "" : "grey";
                             text: {
                                 var txt= qsTr("Mounted as %1").arg(mountpoints.join(", "));
                                 if (isReadOnly) {
-                                    txt += " " + qsTr("[WRITE PROTECTED]")
+                                    txt += " " + qsTr("[WRITE PROTECTED]");
+                                } else if (isSystem) {
+                                    text += " [" + qsTr("SYSTEM") + "]";
```

Comment:
Interesting that `" " +qsTr("[WRITE PROTECTED]")` has the `[ ]` _inside_ the translatable string, but " [" + qsTr("SYSTEM") + "]" has the `[ ]` outside of the translatable string.
(I guess the latter is preferable, but we can't change the former to match without "breaking" a bunch of already-existing translations?)